awx/main/analytics/collectors.py

```diff
@@ -376,13 +376,14 @@ def events_table_partitioned_modified(since, full_path, until, **kwargs):
     return _events_table(since, full_path, until, 'main_jobevent', 'modified', project_job_created=True, **kwargs)
 
 
-@register('unified_jobs_table', '1.2', format='csv', description=_('Data on jobs run'), expensive=four_hour_slicing)
+@register('unified_jobs_table', '1.3', format='csv', description=_('Data on jobs run'), expensive=four_hour_slicing)
 def unified_jobs_table(since, full_path, until, **kwargs):
     unified_job_query = '''COPY (SELECT main_unifiedjob.id,
                                  main_unifiedjob.polymorphic_ctype_id,
                                  django_content_type.model,
                                  main_unifiedjob.organization_id,
                                  main_organization.name as organization_name,
+                                 main_executionenvironment.image as execution_environment_image,
```

Comment:
@AlanCoding  Yeah it would be a new `@register` method. I guess it might be worth to pull the image table if we'd add more data to it. (and removing anything sensitive, e.g. description). Are we able to fetch which registry (Hub or I think we support also other registries?) it comes from? Is that captured in the credential?

Reply:
> Are we able to fetch which registry (Hub or I think we support also other registries?) it comes from?

Not remotely close. There is no line of visibility into what podman does. We don't have a structured way to report _if_ the image was pulled. Because runner has a minimal set of dependencies, it's hard to see where you could even start to develop these abilities. Picture doing a `podman run` command on the CLI, that's what normal jobs do.

Reply:
ok, so we don't nee to complicate this, lets just keep here the image name (with tag I assume) like you have it now :-)

Reply:
@AlanCoding what is database type of the main_executionenvironment.image?
Is some actual db schema available?